.circleci/config.yml

```diff
@@ -18,21 +23,23 @@ jobs:
               curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
             fi
             source $HOME/.cargo/env
-            rustup default nightly
-            rustup update nightly
-            rustup target add wasm32-unknown-unknown --toolchain nightly
+            rustup toolchain update nightly
+            rustup toolchain install nightly-"$(cat NIGHTLY_DATE)"
```

Comment:
why update and then install?
can you `rustup toolchain update nightly` and figure out the latest nightly got installed from update command?

Reply:
good find. I thought there was a bug in rustup where it was missing part of the toolchain, but upon reflection this extra update is unnecessary